oci.go

```diff
@@ -17,11 +17,41 @@ package main
 import (
 	"fmt"
 	"os"
+	"strings"
 	"syscall"
 
 	vc "github.com/containers/virtcontainers"
 )
 
+func envsToEnvVars(envs []string) ([]vc.EnvVar, error) {
```

Comment:
Of course, it makes more sense. I did that to avoid the re-vendoring of virtcontainers, but now that we have this PR#44 merged in virtcontainers, I will rely onto it :)